### PR TITLE
DOC: Morlet wavelet length in tfr_morlet

### LINKED ISSUE #11645: tfr_array_morlet issues with wavelet
### Steps to reproduce

```Python
import numpy as np
from mne.time_frequency import tfr_array_morlet

freqs=np.arange(1, 50, 1)
power = tfr_array_morlet(Input_data, 1000, freqs, n_cycles=7.0, output='avg_power') # Input data is a 70 * 128 * 7000 array
```

### Link to data

I can provide actual data of a single (or couple) channel(s), if necessary.

### Expected results

An array of shape (n_chans, n_freqs, n_times)

### Actual results

ValueError: At least one of the wavelets is longer than the signal. Use a longer signal or shorter wavelets.

The function works ONLY for `n_cycles <= 4`.

### Additional information

Platform:         Windows-10-10.0.22621-SP0
Python:           3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Executable:       C:\Users\user\AppData\Local\Programs\Python\Python310\python.exe
CPU:              Intel64 Family 6 Model 167 Stepping 1, GenuineIntel: 12 cores
Memory:           15.7 GB

mne:              1.3.1
numpy:            1.24.2 {unknown linalg bindings (threadpoolctl module not found: No module named 'threadpoolctl')}
scipy:            1.10.1

### PR DESCRIPTION
**Pull request by ruuskas**
Closes #11645.

This PR adds clarification of the Morlet wavelet temporal window length used with `mne.time_frequency.tfr_array_morlet` and `mne.time_frequency.tfr_morlet`.

### CODE REVIEW

**Comment by larsoner** [comment]
FYI CircleCI will fail with a warning about bool builting linking until https://github.com/mne-tools/mne-python/pull/12068 is merged

**Comment by larsoner** [comment]
Looks like there is a failure in `mne/tests/test_docstring_parameters.py::test_docdict_order`
```
  +  'temporal-window_tfr_morlet_notes',
     'temporal-window_tfr_multitaper_notes',
  -  'temporal-window_tfr_morlet_notes',
```
i.e. it's no longer alphabetical

**Comment by ruuskas** [comment]
The robots seem to be happy, how about the humans? @drammock

File: mne/time_frequency/tfr.py [hunk 1]
```diff
@@ -1040,7 +1039,7 @@ def tfr_array_morlet(
     Notes
     -----
     %(morlet_notes)s
-    %(temporal-window_tfr_notes)s
+    %(temporal-window_tfr_morlet_notes)s
```
Comment (drammock) [inline mne/time_frequency/tfr.py:1042]:
```suggestion

    %(temporal-window_tfr_morlet_notes)s
```

File: mne/utils/docs.py [hunk 2]
```diff
@@ -4431,18 +4429,43 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
...
+``freqs`` and ``n_cycles``, which define the frequencies of interest
+and the number of cycles, respectively. For the time-frequency representation,
+the length of the wavelet is defined such that both tails of
+the wavelet extend five standard deviations from the midpoint of its Gaussian
+envelope and that there is a sample at time zero.
+
+The length of the wavelet is thus :math:`10\times\mathtt{sfreq}\cdot\sigma-1`,
+which is equal to :math:`\frac{5}{\pi} \cdot \frac{\mathtt{n\_cycles} \cdot
+\mathtt{sfreq}}{\mathtt{freqs}} - 1`, where
+:math:`\sigma = \frac{\texttt{n\_cycles}}{2\pi f}` corresponds to the standard
```
Comment (drammock) [inline mne/utils/docs.py:4447]:
```suggestion
:math:`\sigma = \frac{\mathtt{n\_cycles}}{2\pi f}` corresponds to the standard
```
if you use `texttt` the backslash escape gets rendered

File: mne/time_frequency/tfr.py [hunk 3]
```diff
@@ -1121,7 +1120,7 @@ def tfr_multitaper(
 
     Notes
     -----
-    %(temporal-window_tfr_notes)s
+    %(temporal-window_tfr_multitaper_notes)s
```
Comment (drammock) [inline mne/time_frequency/tfr.py:1123]:
```suggestion
    %(temporal-window_tfr_multitaper_notes)s

```
here also we need an extra blank line to separate paragraphs

**Review by drammock — COMMENTED** [review]
very nice!  A few minor formatting tweaks; I'm not 100% sure that all of the extra blank lines are necessary (but at least *some* of them are). Let's double-check the rendered docs again after rebuild.

**Comment by ruuskas** [comment]
Seems that the CIs failing are unrelated @drammock (?)

**Comment by drammock** [comment]
> Seems that the CIs failing are unrelated @drammock (?)

yes definitely unrelated, however the rendering still squashes the paragraphs together; @larsoner is looking into it.

**Review by larsoner — APPROVED** [review]

**Comment by larsoner** [comment]
Marking for merge when green, thanks @ruuskas !

**Pull request merged**